wled00/FX.h

```diff
@@ -809,22 +806,24 @@ class WS2812FX {  // 96 bytes
 #ifndef WLED_DISABLE_2D
     #define WLED_MAX_PANELS 64
     uint8_t
-      hPanels,
-      vPanels;
+      panels;
 
     uint16_t
-      panelH,
-      panelW,
       matrixWidth,
       matrixHeight;
 
     typedef struct panel_bitfield_t {
-      bool bottomStart : 1; // starts at bottom?
-      bool rightStart  : 1; // starts on right?
-      bool vertical    : 1; // is vertical?
-      bool serpentine  : 1; // is serpentine?
+      unsigned int xOffset  : 8; // x offset relative to the top left of matrix in LEDs
```

Comment:
You may want to move these four variables out of bitfield to allow lower CPU overhead.

Reply:
I am still waiting for these 4 to get removed from bitfield.

Reply:
Did it myself.